static/js/task/controllers/task.controller.js

```diff
@@ -74,23 +74,15 @@
 
         function skip() {
             Task.skipTask(self.task_id).then(function success(data, status) {
-                    if (data[1]==200){
```

Comment:
@dmorina auth PR broke error reporting and the upload bug, this is not the right way to handle this use but I added it back.